var/spack/repos/builtin/packages/py-hatchling/package.py

```diff
@@ -15,6 +15,8 @@ class PyHatchling(PythonPackage):
 
     license("MIT", checked_by="tgamblin")
 
+    version("1.25.0", sha256="7064631a512610b52250a4d3ff1bd81551d6d1431c4eb7b72e734df6c74f4262")
```

Comment:
Check the dependencies for the two new versions of py-hatchling; they aren't the same as the versions below